mne/viz/misc.py

```diff
@@ -578,3 +576,92 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
     if show:
         fig.show()
     return fig
+
+
+def plot_phase_amplitude_coupling(phase_bins, normalized_amplitude,
+                                  title=None, show=True):
+    """Plot the phase amplitude plot.
+
+    Parameters
+    ----------
+    phase_bins : array, shape (number of bins + 1,)
+        Binned phase time series of phase modulating signal.
+    normalized_amplitude : ndarray, shape (n_epochs, bin_num)
+        The normalized amplitude values across each bin.
+    title : str, None
+        Title, default is 'Phase amplitude plot'.
+    show : bool
+        Call pyplot.show() at the end. Default is True.
+
+    Returns
+    -------
+    fig : Instance of matplotlib.figure.Figure
+          Figure
+    """
+    if title is None:
+        title = 'Phase amplitude plot'
+    import matplotlib.pyplot as plt
+    for i in range(len(phase_bins) - 1):  # the bins are always more
+        plt.locator_params(axis='x', nbins=4)
+        plt.bar(np.rad2deg(phase_bins[i]), normalized_amplitude[i],
+                width=10, align='edge')
+        plt.xlabel('Phase bins (deg)')
+        plt.ylabel('Normalized Mean Amplitude')
+        plt.title(title)
+
+    if show:
+        plt.show()
+    return
+
+
+def plot_cross_frequency_coupling(times, freqs, traces, ztraces,
+                                  z_threshold, avg, show=True):
+    """Plot cross frequency coupling.
```

Comment:
These docstrings should be a lot more clear. The short description can't just be a string version of the function name. E.g., how does `plot_cross_frequency_coupling` differ from `plot_phase_amplitude_coupling`?